services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: id,
+        cinema_hall_id: int
+) -> None:

```

Comment:
The function `create_movie_session` should not return `None`. It should return the created `MovieSession` instance. Update the return type to `MovieSession`.